saleor/order/models.py

```diff
@@ -143,15 +163,18 @@ def get_subtotal(self):
     def get_total_quantity(self):
         return sum([line.quantity for line in self])
 
-    def can_edit(self):
-        return self.status == OrderStatus.UNFULFILLED
+    def is_draft(self):
+        return self.status == OrderStatus.DRAFT
 
-    def can_fulfill(self):
+    def is_open(self):
         statuses = {OrderStatus.UNFULFILLED, OrderStatus.PARTIALLY_FULFILLED}
         return self.status in statuses
 
     def can_cancel(self):
-        return self.status != OrderStatus.CANCELED
+        return self.status not in {OrderStatus.CANCELED, OrderStatus.DRAFT}
+
+    def can_mark_as_paid(self):
```

Comment:
This will trigger new query when fired, I've seen its used only in template once, can we pass it from the view instead? I think payments should be already prefetched on order detail view

Reply:
True, such helpers are very convenient, but those that perform database queries should be rather avoided.

Reply:
Ok, I'll fix it.